src/components/Startpage/Startpage.tsx

```diff
@@ -0,0 +1,255 @@
+        <div className="cards-wrapper">{
+            popularSearches.map(search =>
+            (
+                <div className="card-wrapper" key={search.url}>
+                    <Link className="disable-link-style" to={search.url}>
+                        <Card className="card">
+                            <Card.Header style={{ height: "100%" }}>
+                                <Card.Title>{search.title}</Card.Title>
+                            </Card.Header>
+                        </Card>
+                    </Link>
+                </div>
+            ))}
+        </div>
+    )
+
+    let endedAuctionsElement = (
+        <div className="cards-wrapper">{
+            endedAuctions.map(getAuctionElement)}
+        </div>
+    )
+
+    let newPlayersElement = (
+        <div className="cards-wrapper">{
+            newPlayers.map(newPlayer =>
+            (
+                <div className="card-wrapper" key={newPlayer.uuid}>
+                    <Link className="disable-link-style" to={`/player/${newPlayer.uuid}`}>
+                        <Card className="card">
+                            <Card.Header style={{ padding: "10px" }}>
+                                <div style={{ float: "left" }}>
+                                    <img crossOrigin="anonymous" src={newPlayer.iconUrl} width="32" height="32" alt="" style={{ marginRight: "5px" }} loading="lazy" />
+                                </div>
+                                {newPlayer.name}
+                            </Card.Header>
+                        </Card>
+                    </Link>
+                </div>
+            ))}
+        </div>
+    )
+
+    let newItemsElement = (
+        <div className="cards-wrapper">{
+            newItems.map(newItem =>
+            (
+                <div className="card-wrapper" key={newItem.tag}>
+                    <Link className="disable-link-style" to={`/item/${newItem.tag}`}>
+                        <Card className="card">
+                            <Card.Header style={{ height: "100%" }}>
+                                <div style={{ float: "left" }}>
+                                    <img crossOrigin="anonymous" src={newItem.iconUrl} width="32" height="32" alt="" style={{ marginRight: "5px" }} loading="lazy" />
+                                    {newItem.name}
+                                </div>
+                            </Card.Header>
+                        </Card>
+                    </Link>
+                </div>
+            ))}
+        </div>
+    )
+
+    //style={{ display: "flex", flexWrap: "wrap", justifyContent: "space-between", alignItems: "stretch" }}
+
+    return (
+        <div className="startpage">
+
+            <div className="startpage-list-element-wrapper">
+                <Card style={{ width: "100%" }}>
+                    <Card.Header>
+                        <Card.Title><AnnouncementIcon/><span style={{ color: "#40ff00" }}> News / Announcements</span></Card.Title>
```

Comment:
Why did you deviate from the mock up here?
The top element should be `Skyblock AH history` (`h1`) followed by text similar to:  
`Browse through 200 million auctions, over two million players and the bazaar of hypixel skyblock`